Makefile-tools.mk

```diff
@@ -7,6 +7,9 @@ MYGOBIN = $(shell go env GOPATH)/bin
 endif
 export PATH := $(MYGOBIN):$(PATH)
 
+# determines whether to run tests that only behave locally; can be overridden by override variable
+export IS_LOCAL = false
```

Comment:
@natasha41575 I know we discussed not changing the Makefiles, but I think this line serves 2 purposes:
1. It makes apparent that `IS_LOCAL` is a make variable users may have a need to pass in, and documents it.
2. By setting the variable to false, we preclude the unintended interference of `IS_LOCAL` being defined in the user's environment for a different use case.

Reply:
SGTM, thanks!